src/bridge/cockpitdisksamples.c

```diff
@@ -148,3 +153,128 @@ cockpit_disk_samples (CockpitSamples *samples)
   g_strfreev (lines);
   g_free (contents);
 }
+
+static gboolean
+read_file (const gchar *path,
+           gchar **contents)
+{
+  g_autoptr(GError) error = NULL;
+  if (!g_file_get_contents (path, contents, NULL, &error))
+    {
+      // do not log following erros:
+      // file does not exist: process died and can no longer be sampled
+      // access denied: bridge is not privileged and can't sample all processes
+      if (!(g_error_matches (error, G_FILE_ERROR, G_FILE_ERROR_NOENT) ||
+          g_error_matches (error, G_FILE_ERROR, G_FILE_ERROR_ACCES)))
```

Comment:
Looks like sometimes [G_FILE_ERROR_FAILED](https://docs.gtk.org/glib/error.FileError.html?q=#failed) is returned...